static/nodejs_server/map_tile.ts

```diff
@@ -190,7 +193,7 @@ export async function getMapChart(
   );
   try {
     const chartData = await fetchData(tileProp);
-    return getMapChartSvg(tileProp, chartData);
+    return getMapChartSvg(tileProp, chartData[0]);
```

Comment:
I don't think chartData is an array right?

Reply:
Great catch! Fixed.